src/pages/Page3.tsx

```diff
@@ -0,0 +1,17 @@
+import MyTabs from "../components/myTabs/MyTabs";
+import { myPageStyle } from "./Page2";
+
+const Page3 = () => {
+  let myPageStyle1 = {
+    ...myPageStyle,
+    alignItems: "top",
+    justifyContent:'left'
+  };
```

Comment:
Use the styled or css api of Emotion.
Go through its docs and then refactor here

Reply:
refactored by Emotion